src/database/mutations/ScheduledItem.ts

```diff
@@ -40,7 +40,13 @@ export async function createScheduledItem(
       createdBy: username,
     },
     include: {
-      approvedItem: true,
+      approvedItem: {
+        include: {
+          authors: {
+            orderBy: [{ sortOrder: 'asc' }],
+          },
+        },
+      },
```

Comment:
Despite _seemingly_ pulling in `authors` from Prisma, integration tests for all `ScheduledItem`-related mutations are failing to find the `authors` prop on the included `approvedItem`.